models/classes/CrudResultsService.php

```diff
@@ -181,4 +188,23 @@ protected function getClassService()
     {
         // TODO: Implement getClassService() method.
     }
+
+    /**
+     * @param $variable
+     * @return array
+     * @throws common_exception_Error
+     */
+    protected function formResource($variable){
```

Comment:
What is the type of `$variable` 
Is it an stdClass?

Reply:
from description it should be `taoResultServer_models_classes_ResponseVariable`. But there is a lot of different implementations.